config/datasets/poesy_refacto.json

```diff
@@ -31,37 +31,11 @@
         "standardize": true,
         "file_format": "npy"
     },
-    "dataset": {
-        "arome": {
-            "members": [
-                0,
-                1, 
-                2,
-                3,
-                4,
-                5,
-                6,
-                7,
-                8,
-                9,
-                10,
-                11,
-                12,
-                13,
-                14,
-                15
-            ],
-            "term": {
-                "start": 0,
-                "end": 45,
-                "timestep": 1
-            },
-            "var": {
-                "TEMPERATURE_2m": {"shortname":"t2m", "filename": "t2m", "level": [2], "typeOfLevel" : "heightAboveGround", "unit" : "K"},
-                "PRECIPITATIONS_DECUM":{"shortname":"tirf", "filename": "rrdecum", "level": [0],"typeOfLevel" : "surface", "unit" : "kg * m **-2"},
-                "WIND.U.PHYS":{"shortname":"u10", "filename": "u","level": [10], "typeOfLevel" : "heightAboveGround", "unit" : "m * s**-1"},
-                "WIND.V.PHYS":{"shortname":"v10", "filename": "v","level": [10], "typeOfLevel" : "heightAboveGround", "unit" : "m * s**-1"}
-                }
-            }
-        }
+    "params": {
+        "t2m" : {"kind" : "input_output", "levels" : [2]},
+        "u10" : {"kind" : "input_output", "levels" : [10]},
+        "v10" : {"kind" : "input_output", "levels" : [10]},
+        "tirf": {"kind" : "input_output", "levels" : [0]}
+    },
+    "members": [0]
```

Comment:
Is the script still running with several members ?

Reply:
Good question !!

Reply:
Yes, it does, tested with 1, 3 and 10 members